src/content/configuration/output.md

```diff
@@ -1133,6 +1134,25 @@ module.exports = {
 };
 ```
 
+## `output.workerChunkLoading`
+
+`string = 'require' | 'import-scripts' | 'async-node' | 'import' | 'universal'` `boolean: false` 
+
+The new option `workerChunkLoading` controls the chunk loading of workers. 
+
+T> The default value of this option changes when `target` is different. For more details, see [`output.workerChunkLoading` default value](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js#L604) and [`target.js`](https://github.com/webpack/webpack/blob/master/lib/config/target.js#L64).
```

Comment:
We have faced numerous issues with linking to LOC on webpack master, code moves too quick. Suggest we note the searchable option in a specified file instead
```suggestion
T> The default value of this option is depending on the `target` setting. For more details, search for `"workerChunkLoading"` : [ in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
```

Reply:
Okay. I see.

Reply:
```suggestion
T> The default value of this option is depending on the `target` setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
```